ulohy_04/Caesar_cipher.py

```diff
@@ -0,0 +1,25 @@
+# Vstup - ošetření, že text bude jen ASCII
+# a klíč jen celé a kladné číslo do 26 (máme 26 znaků)
+while True:
+    text = input("Zadej text, který chceš zakódovat: ")
+    if text.isascii():
+        while True:
+            key = input("Jaký klíč chceš použít? ")
+            if key.isdigit() and int(key) > 0 and int(key) % 26 != 0:
+                break
```

Comment:
Pěkná kontrola :) Ale když zadám prázdný text, program mi vypíše prázdný řádek. To by šlo také ošetřit a vyžadovat vstup nebo vypsat hlášku, že nebylo co šifrovat a už se vůbec neptát na klíč. Která varianta, ti přijde lepší? :)

Reply:
Zařadila jsem podmínku pro prázdný řetězec. Tato situace mě nenapadla ošetřit.